database/Tables/building.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS [dbo].[Buildings];
+GO
+
+CREATE TABLE Buildings (
+    id INT NOT NULL IDENTITY(1,1) PRIMARY KEY,
+    name VARCHAR(255) NOT NULL UNIQUE,
+    code VARCHAR(7)  NOT NULL
```

Comment:
```suggestion
    code VARCHAR(7)  NOT NULL UNIQUE
```

If the code is globally unique, explicit declaration at the database level is good.